src/core/qgsapplication.cpp

```diff
@@ -1687,7 +1687,7 @@ void QgsApplication::registerGdalDriversFromSettings()
   QStringList myList;
   if ( !joinedList.isEmpty() )
   {
-    myList = joinedList.split( ' ' );
+    myList = joinedList.split( QStringLiteral( "," ) );
```

Comment:
This will break existing configuration, right? Probably safer to change the setting key name, and use the old value with the space delimiter only if the newer (comma separated) key doesn't exist

Reply:
Yes, you are right. Fixed according to your suggestion.